### ISSUE TITLE
US-26: As a student I want to enter my start and end points on a building’s floor plan by search

### ISSUE THREAD

**Issue by MrJCipherButtles** [MrJCipherButtles/concordi_around#32]
_Set start and/or end room or facility by search_

**Priority:** Mandatory

**Acceptance Criteria:** When using the search bar, the user can enter a room as a starting or end point.

**Implementation Details:** The search bar will be implemented using the widgets tools available in Flutter. When searching for a room, a list of all available rooms appears (which is a widget Incorporated inside the search bar widget). As the user refines their search, the list of rooms also narrows.

![US-26](https://user-images.githubusercontent.com/45246707/75503833-cdc57580-59a4-11ea-8577-1e3056493526.PNG)

**Comment by DiveshPatel** [comment]
I will be creating a new branch for this feature once the General UI branch is merged into master
